install/ontology/blueprints.rdf

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+        xml:base="http://www.taotesting.com/ontologies/blueprints.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
+>
+    <!-- Class to represent test blueprint -->
+    <!-- Contains identifier, title & content to store additional info -->
+    <rdf:Description rdf:about="http://www.taotesting.com/ontologies/blueprint.rdf#Blueprint">
+        <rdfs:label xml:lang="en-US"><![CDATA[Blueprint]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Blueprint constraints.]]></rdfs:comment>
+        <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#TAOObject"/>
+    </rdf:Description>
+
+    <!-- Blueprint identifier -->
+    <!-- Required -->
+    <!-- Not language dependent -->
+    <rdf:Description rdf:about="http://www.taotesting.com/ontologies/blueprint.rdf#identifier">
```

Comment:
As soon as we get appropriate Solar indexers for #identifier, and #content, I'm happy !